modules/terraform/apply_test.go

```diff
@@ -119,3 +119,36 @@ func TestTgApplyOutput(t *testing.T) {
 	allOutputs := OutputForKeys(t, options, []string{"str", "list", "map"})
 	assert.Equal(t, allOutputs, map[string]interface{}{"str": "str", "list": []interface{}{"a", "b", "c"}, "map": map[string]interface{}{"foo": "bar"}})
 }
+
+func TestIdempotentNochanges(t *testing.T) {
+	t.Parallel()
+
+	testFolder, err := files.CopyTerraformFolderToTemp("../../test/fixtures/terraform-no-error", t.Name())
+	require.NoError(t, err)
+
+	options := &Options{
+		TerraformDir: testFolder,
+		NoColor:      true,
+	}
+
+	ApplyAndIdempotent(t, options)
+}
+
+func TestIdempotentWithchanges(t *testing.T) {
+	t.Parallel()
+
+	testFolder, err := files.CopyTerraformFolderToTemp("../../test/fixtures/terraform-not-idempotent", t.Name())
+	require.NoError(t, err)
+
+	options := &Options{
+		TerraformDir: testFolder,
+		NoColor:      true,
+	}
+
+	out, err := ApplyAndIdempotentE(t, options)
+
+	_ = out
```

Comment:
What is this for?

Reply:
The compiler was complaining about an unused variable. I googled and it said to do this 🤣. 

I changed it to `require.Equal(t, out, "")`